lightkurve/seismology/core.py

```diff
@@ -602,16 +643,70 @@ def estimate_radius(self, teff, numax=None, deltanu=None):
         """
         numax = self._validate_numax(numax)
         deltanu = self._validate_deltanu(deltanu)
+        if teff is None:
+            teff = self.periodogram.meta.get('teff')
+            if teff is None:
+                raise ValueError("You must provide an effective temperature argument (`teff`) to `estimate_radius`,"
+                        "because the Periodogram object does not contain it in its meta data (i.e. `pg.meta['teff']` is missing")
+            else:
+                log.info("Using value for effective temperature from the Kepler Input Catalogue."
+                        "These temperatue values may sometimes differ significantly from modern estimates.",
+                        LightkurveWarning)
```

Comment:
Oops, we need to remove `LightkurveWarning` from all calls to `log.info()`.  The log functions don't take a warning type as an argument.